File: fixture-app-python/discount_test.py
from discount import bulk_discount


def test_bulk_discount_applies_at_exactly_ten_units():
    # 10 units at 100 each should get the 10% bulk discount -> 900
    assert bulk_discount(100, 10) == 900


def test_bulk_discount_applies_above_ten_units():
    assert bulk_discount(100, 11) == 990


def test_bulk_discount_not_applied_below_ten_units():
    assert bulk_discount(100, 9) == 900

File: fixture-app-rust/tests/discount_test.rs
use revi_fixture_app_rust::discount::bulk_discount;

#[test]
fn applies_bulk_discount_at_exactly_ten_units() {
    // Orders of exactly 10 units should receive the 10% bulk discount.
    assert_eq!(bulk_discount(100, 10), 900);
}

#[test]
fn applies_bulk_discount_above_ten_units() {
    assert_eq!(bulk_discount(100, 11), 990);
}

#[test]
fn no_discount_below_ten_units() {
    assert_eq!(bulk_discount(50, 9), 450);
}
